src/test/ui/precise_pointer_size_matching.rs

```diff
@@ -1,3 +1,10 @@
+// normalize-stderr-32bit: "-2147483648isize" -> "$$ISIZE_MIN"
+// normalize-stderr-64bit: "-9223372036854775808isize" -> "$$ISIZE_MIN"
+// normalize-stderr-32bit: "2147483647isize" -> "$$ISIZE_MAX"
+// normalize-stderr-64bit: "9223372036854775807isize" -> "$$ISIZE_MAX"
+// normalize-stderr-32bit: "4294967295usize" -> "$$USIZE_MAX"
+// normalize-stderr-64bit: "18446744073709551615usize" -> "$$USIZE_MAX"
```

Comment:
I guess this is neat but it's not enough for 16-bit targets.

Reply:
compilertest itself doesn't recognize 16-bit targets 🙁 

https://github.com/rust-lang/rust/blob/367e783e6f66a4dba32decdc68a262953d2f3f1b/src/tools/compiletest/src/util.rs#L103-L109